src/main/java/sirius/web/dispatch/HelpDispatcher.java

```diff
@@ -38,14 +35,29 @@ public boolean preDispatch(WebContext ctx) throws Exception {
     }
 
     @Override
+    public int getPriority() {
+        return PriorityCollector.DEFAULT_PRIORITY + 100;
+    }
+
+    @Override
     public boolean dispatch(WebContext ctx) throws Exception {
         if (!ctx.getRequest().uri().startsWith("/help") || HttpMethod.GET != ctx.getRequest().method()) {
             return false;
         }
         String uri = ctx.getRequestedURI();
+        String lang = "";
```

Comment:
I just have this param to construct the "home"-link of the help system.

My thinking was:
I have "de" when I call "/help/de/some-page" and can construct the home-link "/help/de".
When there is no language configured I don't want a "home"-link of "/help/defaultLang", but to "/help".

Your opinion?